### PR TITLE
Packages zip and unzip

### PR DESCRIPTION
**Pull request by jerowe**
These were in @msarahan 's PR, and are basically his recipe, but I would like to move this allow a bit. ;-)

### CODE REVIEW

**Comment by jerowe** [comment]
The original work for this PR is here: https://github.com/conda-forge/staged-recipes/pull/1533#

ping @jakirkham

File: recipes/unzip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% set version="6.0" %}
+{% set url_version="60" %}
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Maybe `"60"` -> `version.replace(".", "")`.

File: recipes/zip/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% set version="3.0" %}
+{% set url_version="30" %}
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Similar change as above.

File: recipes/unzip/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  skip: True  # [win]
+  script: make -f unix/Makefile generic && mv unzip $PREFIX/bin
+
+test:
+  commands:
+    - unzip -h
+
+about:
+  home: http://www.info-zip.org/
+  license: MIT
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Should be `BSD-like` or something more descriptive if you can find it.

File: recipes/zip/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  skip: True  # [not linux]
+  script: make -f unix/Makefile generic && mv zip $PREFIX/bin
+
+test:
+  commands:
+    - zip -h
+
+about:
+  home: http://www.info-zip.org/
+  license: MIT
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Same as above.

File: recipes/unzip/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  license_file: LICENSE
+  summary: simple program for unzipping files
+
+extra:
+  recipe-maintainers:
+    - msarahan
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Would you like to be a maintainer?

File: recipes/zip/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  license_file: LICENSE
+  summary: simple program for unzipping files
+
+extra:
+  recipe-maintainers:
+    - msarahan
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Would you like to be a maintainer?

**Comment by jerowe** [comment]
This is the license page for unzip

http://www.info-zip.org/pub/infozip/license.html

File: recipes/unzip/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: xref: http://www.info-zip.org/license.html

File: recipes/zip/meta.yaml (hunk 4 above)
Comment (jakirkham) [inline recipes/zip/meta.yaml]: xref: http://www.info-zip.org/license.html

File: recipes/zip/meta.yaml (hunk 6 above)
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Done below.

File: recipes/unzip/meta.yaml (hunk 5 above)
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Done below.

File: recipes/zip/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  name: zip
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.info-zip.org/pub/infozip/src/zip{{ version.replace(".", "") }}.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Maybe just skip Windows. Could be useful on OS X too.

**Comment by jakirkham** [comment]
Prefacing with IANAL. The license AFAICT reads basically like a BSD 3-Clause license. They just add in some stuff to protect particular software names or avoid offering any support for derivative works. We have used BSD-like before in case like this so it should be ok here. Plus we are providing the license file for people that wish to know more.

**Comment by jakirkham** [comment]
Also feel free to add `[skip appveyor]` in commit messages as we are not building for Windows.

**Comment by jerowe** [comment]
I'm not very familiar with licensing models. 

What should I put. BSD-3?

**Comment by jakirkham** [comment]
IMHO BSD-like. Alternatively we can call it ZIP and maybe give it a year.

File: recipes/example/meta.yaml [hunk 8]
```diff
@@ -21,6 +21,8 @@ source:
 build:
   number: 0
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Note to others: This was already added in `master` with PR ( https://github.com/conda-forge/staged-recipes/pull/1632 ). So this really isn't an addition, but some issue rendering the diff.

File: recipes/example/meta.yaml [hunk 9]
```diff
@@ -60,6 +62,7 @@ about:
 
 extra:
   recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Note to others: Same as above. Comes from PR ( https://github.com/conda-forge/staged-recipes/pull/1632 ) already merged into `master`.

**Comment by jakirkham** [comment]
Thanks @msarahan and @jerowe. Let's get these out there. 😄

**Comment by jerowe** [comment]
Thanks, @jakirkham ! And thanks @msarahan for getting this started!

**Comment by jakirkham** [comment]
Soon you should get an email that invites you to join conda-forge, @jerowe. Once accepted you will be added to a team with the same names as these recipes. Those will give you permissions on the feedstocks (repos) for these recipes.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make.

Please let us know if you have any questions and welcome to conda-forge. :)

**Pull request merged**